sidecar-advanced-custom/README.md

```diff
@@ -0,0 +1,15 @@
+# Creating a Yearbook in the Student's side pane
+
+## Prerequisites
+1. This tutorial requires to have the Professor M package intalled. Please find the instructions [here](https://github.com/sugarcrm/uncon/tree/2017/ProfessorM)
```

Comment:
Please change to:

This tutorial requires you to have the Professor M package installed.